glean-core/metrics.yaml

```diff
@@ -75,6 +75,26 @@ glean.internal.metrics:
       - glean-team@mozilla.com
     expires: never
 
+
+  windows_build_number:
+    type: string
+    lifetime: application
+    send_in_pings:
+      - glean_client_info
+    description: |
+      The optional user-visible version of the Windows Build Number
+      (e.g. "1.2.3"). If the version detection fails, this metric gets set
+      to `Unknown`.
```

Comment:
windows build numbers are _not_ "1.2.3".
I argue we do not set it to `Unknown` but don't set it at all if we can't detect it (as we obviously can't on non-windows)